pkg/alerts/nrql_conditions.go

```diff
@@ -172,11 +224,22 @@ type NrqlConditionTerm struct {
 // NrqlConditionQuery represents the NRQL query object returned in a NerdGraph response object.
 type NrqlConditionQuery struct {
 	Query            string `json:"query,omitempty"`
-	EvaluationOffset int    `json:"evaluationOffset,omitempty"`
+	EvaluationOffset *int   `json:"evaluationOffset,omitempty"`
```

Comment:
Updated to be a pointer.  This is necessary because users must be able to null out `EvalutionOffset` when updating an existing condition to remove `EvaluationOffset` and replace with `AggregationMethod`.  

This is required because `EvaluationOffset` and `AggregationMethod` are mutually exclusive.

Reply:
I believe this will require a Major Version release.  Open to feedback!